pkg/services/publicdashboards/service/service.go

```diff
@@ -273,17 +358,30 @@ func (pd *PublicDashboardServiceImpl) buildMetricRequest(ctx context.Context, da
 func (pd *PublicDashboardServiceImpl) BuildAnonymousUser(ctx context.Context, dashboard *models.Dashboard) (*user.SignedInUser, error) {
```

Comment:
I think we need to extend test coverage for this method to include annotations scopes